src/v2/Apps/Artwork/Utils/lotIsClosed.ts

```diff
@@ -1,3 +1,9 @@
 export const lotIsClosed = (sale, saleArtwork): boolean => {
-  return !sale || !saleArtwork || sale?.is_closed || !!saleArtwork?.endedAt
+  return (
+    !sale ||
+    !saleArtwork ||
+    sale?.is_closed ||
+    sale?.isClosed ||
```

Comment:
This helper is getting used in a few components, one which aliased `is_closed` for another purpose, and one with `isClosed`. It seems fine to throw that kind of thing in this helper rather than untangle the existing `is_closed` alias from its usage.